test/sqllogictest/types.slt

```diff
@@ -1084,3 +1084,7 @@ SELECT 'table_with_id'::regclass
 # fully qualified names are not yet supported
 statement error object "test.table_with_id" does not exist
 SELECT 'test.table_with_id'::regclass
+
+# over qualified types should return an error
+query error qualified name did not have between 1 and 3 components
+SELECT 'true'::universe.database.schema.bool
```

Comment:
This seems like an odd place for this test because it's equally as applicable for all objects in the catalog. I'd either consolidate this test to a single place or (not recommended) distribute it everywhere it should be tested.

Reply:
Also moved this to `name_resolution.slt`, which is probably more appropriate.